### PR TITLE
add wwid as a parameter for refresh_bootmap

### PR DESCRIPTION
**Pull request by dongyanyang**
Before this change, the wwid is get from the scsi_id cmd on a single path after we find
a valid disk show up on the compute node.

But in some error case that the refreshbootmap may fail in very early stage before we find a valid path, eg
when failed to online some FCPs, or no valid path found on any FCPs, in these cases, the wwid maybe "". Then
when doing cleanup at the exit of refreshbootmap, we cann't get the multipath map name (eg, mpatha) from wwid and
then we cann't do the cleanup of the mpath and disk.

so this change would accept wwid as a parameter and use it directly in the cleanup.

Signed-off-by: dyyang <dyyang@cn.ibm.com>

### CODE REVIEW

**Comment by dongyanyang** [comment]
Function Tested.
@jackydalong @bjhuangr please help review. Thx!

File: zthin-parts/zthin/bin/refresh_bootmap [hunk 1]
```diff
@@ -969,16 +979,23 @@ function refreshFCPBootmap {
...
   wwpn=$(echo ${wwpn} | tr '[:upper:]' '[:lower:]')
+  wwid=$(echo ${wwid} | tr '[:upper:]' '[:lower:]')
 
   # Add 0x to lun
   lun=0x$lun
+  # The disk path in Linux layer will have a '3' as prefix as shown in following example:
+  # /dev/disk/by-id/dm-uuid-part1-mpath-3600507640083826de00000000000605b 
+  # so, append a '3' as a prefix to the input wwid
+  if [[ "$wwid" != "" ]]; then
+      wwid="3"$wwid
```
Comment (bjhuangr) [inline zthin-parts/zthin/bin/refresh_bootmap:990]: "3$wwid" ?

File: zvmsdk/api.py [hunk 2]
```diff
@@ -1661,12 +1661,14 @@ def volume_attach(self, connection_info):
         self._volumeop.attach_volume_to_instance(connection_info)
 
     def volume_refresh_bootmap(self, fcpchannels, wwpns, lun,
+                               wwid='',
```
Comment (bjhuangr) [inline zvmsdk/api.py:1664]: As you have increased waiting time of disk show up in refreshbootmap, do you consider to increase CONF.volume.refresh_bootmap_timeout default value accordingly ?

Comment (dongyanyang) [reply zvmsdk/api.py:1664]: yes, I will. I planned to increase in another new PR. but agree better to commit in same PR.

File: zthin-parts/zthin/bin/refresh_bootmap (hunk 1 above)
Comment (dongyanyang) [reply zthin-parts/zthin/bin/refresh_bootmap:990]: yes, see the example in the comment before this line of code.

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**